paddle/fluid/framework/tensor_util.h

```diff
@@ -24,10 +24,11 @@ namespace paddle {
 namespace framework {
 
 void TensorCopy(const Tensor& src, const platform::Place& dst_place,
```

Comment:
Now that we have `TensorCopySync`, I think we should have `TensorCopyAsync` correspondingly, it should be explicit.

Reply:
I agree. The `TensorCopy` should be `TensorCopyAsync`. However, this modification might be a heavy job for `TensorCopy` is widely used in our current code. Maybe it's better to leave it to further PRs.